package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "signature_pad",
+  "name": "@ptgs/signature_pad",
```

Comment:
Remove these changes

Reply:
Ah sorry, these were only meant to be in my main so I could publish in the meantime. Will fix up.